fdbcli/QuotaCommand.actor.cpp

```diff
@@ -103,8 +103,23 @@ ACTOR Future<Void> setQuota(Reference<IDatabase> db, TransactionTag tag, LimitTy
 	}
 }
 
+ACTOR Future<Void> clearQuota(Reference<IDatabase> db, TransactionTag tag) {
+	state Reference<ITransaction> tr = db->createTransaction();
+	state Key key = tag.withPrefix(tagQuotaPrefix);
+	loop {
+		tr->setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
+		try {
+			tr->clear(ThrottleApi::getTagQuotaKey(tag));
```

Comment:
Should this be `key` instead of `tag` to account for the prefix?

Reply:
`ThrottleApi::getTagQuotaKey(tag)` returns the same as `key`. I removed `key` to avoid this unnecessary confusion.